docs/pages/docs/index.mdx

```diff
@@ -27,4 +27,4 @@ Most importantly, unlike other build systems, Turborepo is designed to be increm
 - **Task pipelines**: Define the relationships between your tasks and then let Turborepo optimize what to build and when.
 - **Pruned subsets**: Speed up PaaS deploys by generating a subset of your monorepo with only what's needed to build a specific target.
 - **Convention-based config**: Reduce complexity through convention. Fan out configuration with just a few lines of JSON.
-- **Profile in your browser**: Generate build profiles and view them in a browser to understand which tasks are taking the longest.
+- **Profile in your browser**: Generate build profiles in the chrometracing format and view them in a browser to understand which tasks are taking the longest.
```

Comment:
We should link to something here. I didn't find a canonical place to link to in a quick search, but if you know of something that would be good.

Reply:
I also didn't find anything. 🤷‍♂️ Blog post?